app/population/index.js

```diff
@@ -1,20 +1,30 @@
 const router = require('express').Router()
+const dailyRouter = require('express').Router({ mergeParams: true })
 
 const {
   setContext,
   setDateRange,
   setPagination,
 } = require('../../common/middleware/collection')
 
-const { BASE_PATH, MOUNTPATH } = require('./constants')
-const { dashboard } = require('./controllers')
-const { setResultsAsPopulationTable, redirectBaseUrl } = require('./middleware')
+const { BASE_PATH, MOUNTPATH, DAILY_PATH } = require('./constants')
+const { dashboard, daily } = require('./controllers')
+const {
+  redirectBaseUrl,
+  setPopulationId,
+  setResultsAsPopulationTable,
+} = require('./middleware')
 
 router.param('date', setDateRange)
+router.param('locationId', setPopulationId)
 
 router.get('/', redirectBaseUrl)
 
-router.use(
+dailyRouter.get('/', setContext('population'), daily)
+
+router.use(DAILY_PATH, dailyRouter)
+
+router.get(
   BASE_PATH,
   setContext('population'),
```

Comment:
Obviously not necessarily related to this specific change but I wonder if `setContext` is serving any purpose on this route either?

Reply:
All of this `population` section can be revisited and refactored once the development patterns have become a little clearer.

The main population grid is definitely not part of this PR